code/game/say.dm

```diff
@@ -38,10 +38,25 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	return 1
 
 /atom/movable/proc/send_speech(message, range = 7, obj/source = src, bubble_type, list/spans, datum/language/message_language = null, list/message_mods = list())
+	// TTS generation
+	if(!isobserver(src)) // Ghosts can shut the fuck up please
```

Comment:
probably could use a better check, like invisibility or similar (would allow for admin funny and less edgecases)

Reply:
ghosts being un-invisible doesnt mean you can hear them

Reply:
why not?

Reply:
wizard rounds, spectral blade, you can't hear what those ghosts have to say and it would be a gateway to meta info

Reply:
That’s the point

Reply:
what